Websites/perf.webkit.org/public/v3/models/test-group.js

```diff
@@ -160,6 +158,16 @@ class TestGroup extends LabeledObject {
         return this._orderedBuildRequests().filter((request) => request.commitSet() == commitSet);
     }
 
+    testParameterSetForCommitSet(commitSet)
+    {
+        return this.requestsForCommitSet(commitSet)?.[0]?.testParameterSet();
+    }
+
+    requestedTestParameterSets()
+    {
+        return this.requestedCommitSets().map(commitSet => this.requestsForCommitSet(commitSet)[0].testParameterSet());
```

Comment:
Nit: This snippet does not chain optionals and the one in `testParameterSetForCommitSet()` above does - should this also be using optionals in case `requestsForCommitSet` can't find the commitSet requested? (or alternatively - can we reuse `testParameterSetForCommitSet` here?)